### PR TITLE
font edited

### PR DESCRIPTION
**Pull request by faraz-2021**

### CODE REVIEW

File: app.json [hunk 1]
```diff
@@ -1,11 +1,10 @@
 {
   "expo": {
     "name": "my-project",
-    "description":"my first todo app",
+    "description": "my first todo app",
     "slug": "my-project",
     "version": "1.0.0",
     "orientation": "portrait",
-    "sdkVersion": "36.0.0",
```
Comment (priyarajputexc) [inline app.json:8]: you removed sdk version?
just need to update it

File: src/accessFont.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  assets: ["./assets/fonts"],
+};
```
Comment (priyarajputexc) [inline src/accessFont.js]: do we need this?

File: package.json [hunk 3]
```diff
@@ -8,22 +8,24 @@
     "eject": "expo eject"
   },
   "dependencies": {
-    "@react-native-community/masked-view": "0.1.10",
+    "@expo-google-fonts/nunito-sans": "^0.1.0",
```
Comment (priyarajputexc) [inline package.json]: why this?

File: app.json (hunk 1 above)
Comment (faraz-2021) [reply app.json:8]: mam it had been updted and removed directly by the exxpo itself

File: package.json (hunk 3 above)
Comment (priyarajputexc) [reply package.json]: still not removed?

File: app.json (hunk 1 above)
Comment (priyarajputexc) [reply app.json:8]: ok

File: package.json (hunk 3 above)
Comment (faraz-2021) [reply package.json]: removing mam

**Pull request merged**